.travis.yml

```diff
@@ -35,4 +25,15 @@ matrix:
     - php: 5.5
       env: SYMFONY_VERSION=2.5.*
     - php: 5.5
-      env: SYMFONY_VERSION=dev-master
+      env: SYMFONY_VERSION=2.6.*
+    - php: 5.5
+      env: SYMFONY_VERSION='2.7.*@dev'
+
+before_script:
+  - composer self-update
+  - composer require symfony/framework-bundle:${SYMFONY_VERSION}
```

Comment:
this does not actually test against Symfony 2.3, given that Frameworkbundle 2.3 is compatible with HttpFoundation 2.7 for instance